app/src/main/res/values/strings.xml

```diff
@@ -182,8 +182,8 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="key_tab">Tabulator</string>
     <string name="start_on_bottom">Start on bottom</string>
     <string name="start_on_bottom_when_loading_document">Upon loading a document, position cursor at its end</string>
-    <string name="filemanager_doesnot_supply_required_data__appspecific">Markor only allows to edit local, offline available, documents and support file managers which supply the full filepath including filename. Apparently the app didn\'t supply all required information and can\'t be used with Markor :(.</string>
-    <string name="sync_to_local_folder_notice">Notice: Sync apps may place files in custom folders, open them by changing Markor\'s document folder accordingly, or in another file manager.</string>
+    <string name="filemanager_doesnot_supply_required_data__appspecific">Markor can only edit local, offline available, documents, and does not support file managers that do not provide the full file path to Markor. This is a bug with the file manager, and not with Markor.</string>
```

Comment:
It's not an bug of a filemanager, they are free to provide it or not. Most do, some do not.